README.md

```diff
@@ -182,10 +182,18 @@ All targets except `default` will have the target name prefixed to the tags - eg
 
 ### balena
 
-If a `balena.yml` file is found in the root of the repository Flowzone will attempt to push draft releases to your application slug(s) and finalize on merge.
+If a `balena.yml` file is found in the root of the repository Flowzone will attempt to push draft releases to your fleet's slug(s) and finalize on merge.
+
+This will **require** your organization or your repository to have an API key secret named as `BALENA_API_KEY` set in GitHub. If you intend to set an org level secret then make sure the API key is valid for all repositories in that organization. A repository level secret will override an organization level secret. This API key will be used to login into balena and push draft releases to your fleet. 
```

Comment:
```suggestion
This will **require** either your organization or your repository to have a balenaCloud API key set as a secret named `BALENA_API_KEY`. If you intend to set the secret at the org level then make sure that the API key is valid for all repositories in that organization. A repository-level secret will override an organization-level secret. This API key will be used to login into balena-cli and push draft releases to your fleet in balenaCloud.  
```

Reply:
Nothing like a bit of self-review.